### ISSUE TITLE
Alum profiles

### ISSUE THREAD

**Issue by schnej7** [RPI-UPE/rpi-upe.github.com#32]
I know I sound like a broken record, but I think adding an alums profile page would be good for 2 reasons.

1) Getting alum more involved will provide more job opportunities.

2) The page will convey the success of UPE members and lead to more applicants.

**Comment by TManzini** [comment]
I totally agree, we are going to be talking about this at the next meeting (What we want to appear and all that). 

Progress is happening!

**Comment by jrock08** [comment]
Another old person chiming in here. I like this idea a lot and think something like 1 line per alum, Name - Job (with the option for it to all be a hyperlink to a personal page) would be a sweet addition.

**Comment by schnej7** [comment]
Also, you should include some way for alums to sign up to be listed on that page.  Maybe a simple mailto will suffice.
